alpaca/Dockerfile

```diff
@@ -29,7 +29,6 @@ RUN bin/start && \
 
 # Triple indexing
 RUN bin/start && \
-    bin/client -r 10 -d 5  "feature:install fcrepo-indexing-triplestore" && \
```

Comment:
This service is keeping me from being able to run on Arm for some reason I cannot sort out (Karaf makes me wanna pull my hair out), is it not needed on main either?